site/docs/numbers/porting/csrLookupTool.mdx

```diff
@@ -32,47 +32,18 @@ This walks through how to programmatically lookup Customer Service Records (CSRs
 
 ## Polling vs. Webhooks
 
-CSR Orders in the Bandwidth Dashboard are asynchronous when creating an "order". The orders are then processed and the order status is updated asynchronously.  Bandwidth recommends configuring your account with a subscription instead of polling the order resource for `<OrderStatus>`.
+CSR Orders in the Bandwidth Dashboard are asynchronous when creating an "order". The orders are then processed and the order status is updated asynchronously. Bandwidth recommends configuring your account with a subscription instead of polling the order resource for `<OrderStatus>`.
```

Comment:
```suggestion
CSR Orders in the Bandwidth Dashboard are asynchronous when creating an "order". The orders are then processed and the order status is updated. Bandwidth recommends configuring your account with a subscription instead of polling the order resource for `<OrderStatus>`.
```